src/java.desktop/share/classes/javax/swing/JComponent.java

```diff
@@ -1739,10 +1739,13 @@ public Dimension getPreferredSize() {
 
     /**
      * Sets the maximum size of this component to a constant
-     * value.  Subsequent calls to <code>getMaximumSize</code> will always
+     * value.  Subsequent calls to {@code getMaximumSize} will always
      * return this value; the component's UI will not be asked
-     * to compute it.  Setting the maximum size to <code>null</code>
+     * to compute it. Setting the maximum size to {@code null}
      * restores the default behavior.
+     * <p>
+     * Subclasses may choose to override this by returning its own maximum size
+     * in its {@code getMaximumSize} method.
```

Comment:
```suggestion
     * Subclasses may choose to override this by returning their own maximum size
     * in the {@code getMaximumSize} method.
```
“Subclasses” is plural and therefore requires plural possessive pronoun.